core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -106,12 +107,19 @@ public KafkaStore(SchemaRegistryConfig config,
     this.zkUtils = ZkUtils.apply(
         kafkaClusterZkUrl, zkSessionTimeoutMs, zkSessionTimeoutMs,
         JaasUtils.isZkSecurityEnabled());
-    this.kafkaTopicReader =
-        new KafkaStoreReaderThread<>(zkUtils, kafkaClusterZkUrl, topic, groupId,
-                                         Integer.MIN_VALUE, this.storeUpdateHandler,
-                                         serializer, this.localStore, this.noopKey);
     this.brokerSeq = zkUtils.getAllBrokersInCluster();
 
+    List<Broker> brokers = JavaConversions.seqAsJavaList(brokerSeq);
+    StringBuilder sb = new StringBuilder();
+    for (Broker broker : brokers) {
+      for(EndPoint ep : JavaConversions.asJavaCollection(broker.endPoints().values())) {
```

Comment:
Presumably this patch is a step in the direction of enabling security (yay!). But while we still really only support plaintext, we may need to think about the fact that collecting every endpoint, as in this block of code, would cause us to include non-plaintext endpoints in our list.